Source/NNSupportFunctions/arm_nn_mat_mult_nt_t_s32.c

```diff
@@ -0,0 +1,157 @@
+/*
+ * SPDX-FileCopyrightText: Copyright 2023 Arm Limited and/or its affiliates <open-source-office@arm.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* ----------------------------------------------------------------------
+ * Project:      CMSIS NN Library
+ * Title:        arm_nn_mat_mult_s8_nt_t_s32
+ * Description:  Matrix multiplication support function with the right-hand-side (rhs) matrix transposed
+ *
+ * $Date:        5 October 2023
+ * $Revision:    V.1.0.0
+ *
+ * Target :  Arm(R) M-Profile Architecture
+ *
+ * -------------------------------------------------------------------- */
+
+#include "arm_nnsupportfunctions.h"
+
+/**
+ * @ingroup groupSupport
+ */
+
+/**
+ * @addtogroup supportConvolution
+ * @{
+ */
+
+/*
+ * s32 matrix multiplication with the right-hand-side matrix transposed
+ *
+ * Refer header file for details.
+ *
+ */
+arm_cmsis_nn_status arm_nn_mat_mult_nt_t_s32(const int8_t *lhs,
```

Comment:
The naming convention used here is primarily of the input and output when they are the same.. and if they are different like in arm_nn_lstm_calculate_gate_s8_s16.c, then it is of input followed by the output. Following that, could you rename this for a sufix of s8_s32 instead?

Reply:
Yes done